pkg/build/netbsd.go

```diff
@@ -33,29 +36,33 @@ no options SVS
 	if err := osutil.WriteFile(filepath.Join(confDir, kernelName), conf); err != nil {
 		return err
 	}
-
 	// Build tools before building kernel
 	if _, err := osutil.RunCmd(10*time.Minute, kernelDir, "./build.sh", "-m", targetArch,
-		"-U", "-j"+strconv.Itoa(runtime.NumCPU()), "tools"); err != nil {
+		"-U", "-u", "-j"+strconv.Itoa(runtime.NumCPU()), "tools"); err != nil {
 		return extractRootCause(err)
 	}
 
 	// Build kernel
 	if _, err := osutil.RunCmd(10*time.Minute, kernelDir, "./build.sh", "-m", targetArch,
-		"-U", "-j"+strconv.Itoa(runtime.NumCPU()), "kernel="+kernelName); err != nil {
+		"-U", "-u", "-j"+strconv.Itoa(runtime.NumCPU()), "kernel="+kernelName); err != nil {
 		return extractRootCause(err)
 	}
-
-	for _, s := range []struct{ dir, src, dst string }{
-		{compileDir, "netbsd", "kernel"},
-		{compileDir, "netbsd.gdb", "netbsd.gdb"},
+	for _, s := range []struct{ dir, obj string }{
+		{compileDir, "netbsd"},
+		{compileDir, "netbsd.gdb"},
+		{userspaceDir, "image"},
+		{userspaceDir, "key"},
 	} {
-		fullSrc := filepath.Join(s.dir, s.src)
-		fullDst := filepath.Join(outputDir, s.dst)
+		fullSrc := filepath.Join(s.dir, s.obj)
+		fullDst := filepath.Join(outputDir, s.obj)
 		if err := osutil.CopyFile(fullSrc, fullDst); err != nil {
 			return fmt.Errorf("failed to copy %v -> %v: %v", fullSrc, fullDst, err)
 		}
 	}
+
+	if vmType == "gce" {
```

Comment:
We need kernel in the image for "qemu" too, because Kamil said qemu does not support -kernel flag for netbsd, so we won't be able to use injected kernel.
Or remove "qemu" from the switch in pkg/build/build.go as unsupported.

Reply:
What is the use of running sys-ci with the qemu option? Is it intended to help people run sys-ci by themselves?

Reply:
I guess we can remove the qemu flag from pkg/build/build.go

Reply:
For NetBSD/amd64 we need multiboot booting through 32-bit mode.. we got a developer to port multiboot for Xen purposes very recently and I've prompted him about `qemu -kernel` feature to be handled, but so far no echo (and it wasn't compatible with qemu -kernel).

Another option is to encode bootloader into qemu for NetBSD (similarly to Linux booting process of x86_64), but it's rather the last resort solution and MULTIBOOT is useful anyway, not just for qemu (and not just for Xen).

Reply:
I have currently made it 
+ if vmType == "gce" || vmType == "qemu" 

we can probably change it back to "gce" only once we get the multi boot enabled